consequence_prediction/vep_mapping_pipeline/consequence_mapping.py

```diff
@@ -69,11 +71,35 @@ def query_vep(variants, search_distance):
     return result.json()
 
 
+@lru_cache
+@retry(tries=10, delay=5, backoff=1.2, jitter=(1, 3), logger=logger)
+def query_consequence_types():
+    url = 'https://rest.ensembl.org/info/variation/consequence_types?content-type=application/json&rank=1'
+    result = requests.get(url)
+    result.raise_for_status()
+    return result.json()
+
+
+def get_so_accessions():
+    consequence_type_results = query_consequence_types()
+    return {conseq['SO_term']: conseq['SO_accession'] for conseq in consequence_type_results}
```

Comment:
Should we make use of other fields too for documentation? As in a ranked table with all this information (name, description...) for each of our releases. That way the consequences could be traceable. Specially since the URL seems to be static

Reply:
I'm not sure the usefulness of this, we can ask if OT is interested other details.  Everything should be fully documented in [Sequence Ontology](http://www.sequenceontology.org/) via the accession anyway though.

Reply:
I see, fine then 👍